taskcluster/app_services_taskgraph/job.py

```diff
@@ -0,0 +1,176 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This file is a copy-paste from https://github.com/mozilla-mobile/fenix/blob/master/taskcluster/fenix_taskgraph/job.py with the SDK part commented out for now. I left a TODO to address later.